spectrochempy/analysis/mcrals.py

```diff
@@ -1013,20 +1130,36 @@ def St(self):
 
     @property
     @_wrap_ndarray_output_to_nddataset(units=None, title=None, typex="components")
-    def C_hard(self):
+    def C_constrained(self):
         """
-        The hard concentration profiles.
+        The constrained concentration profiles, i.e. after applying the hard and soft constraints.
         """
         return self._outfit[2]
 
+    @property
+    @deprecated(replace="C_constrained")
+    def C_hard(self):
+        """
+        Deprecated. Equivalent to `C_constrained`.
+        """
+        return self.C_constrained
+
     @property
     @_wrap_ndarray_output_to_nddataset(units=None, title=None, typey="components")
-    def St_soft(self):
+    def St_unconstrained(self):
         """
-        The soft spectra profiles.
+        The soft spectra profiles, obtained after solving $C_{\textrm{constrained}} \cdot St = X$ for $St$.
         """
```

Comment:
you cannot use $ for math in rst docstring (this is not markdown!).

Look the result
<img width="728" alt="image" src="https://user-images.githubusercontent.com/2428474/235283777-53ebd31c-b7b6-4df9-a64a-18403cb9604d.png">